docs/point_api_spec.md

```diff
@@ -0,0 +1,69 @@
+# Address Point API Spec
+
+## Purpose
+This document describes the Address Point Geocoding API, serving as a specification or "contract" for other users of this service.
+In particular, it describes the endpoints that this service exposes, as well as the structure and format of the data that is exchanged when using this service.
+
+### Protocols and data formats
+The Address Point Geocoding Service exposes data through HTTP, accepting and returing data in JSON format. 
+In particular, all requests need to specify their content type as appropriate:
+
+```
+Content-Type: application/json
+```
+
+Requests that don't specify their content type will be rejected
+
+### API Endpoints
+
+`GET  /status`
+
+Returns a message with current status and date
+
+```json
+{
+    "status": "OK",
+    "time": "Wed Feb 18 10:30:55 EST 2015"
+}
+```
+
+`POST /address/point`
+
+Payload: JSON object with the following structure:
+
+* id: Identifier for the record (Integer)
+* address: Address to geocode (String)
+
+Example:
+
+```json
+{
+  "id": 1,
+  "address": "16410 N AR 94 Hwy Pea Ridge 72751 AR"
+}
+```
+
+Return: GeoJSON object that includes the id sent in the request as a field, and the point that resulted from geocoding, as well as other attributes including the address that was found.
+
+
+```json
+{
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [
+            -94.15513718509897,
+            36.48036108322251,
+            0
```

Comment:
Will we be using the altitude coordinate?  Reading the GeoJSON spec, it sounds like its optional:

>A position is represented by an array of numbers. There must be at least two elements, and may be more. The order of elements must follow x, y, z order (easting, northing, altitude for coordinates in a projected coordinate reference system, or longitude, latitude, altitude for coordinates in a geographic coordinate reference system). Any number of additional elements are allowed -- interpretation and meaning of additional elements is beyond the scope of this specification.

If we're not using it, is it best practice to leave it off or set it to zero explicitly?  I also notice that it is not included in the [point data spec](https://github.com/cfpb/grasshopper/pull/25/files#diff-a5ff48188b4f707a6ad63ac1f67b06d2R24).  Should that be consistent?